lib/app/doc/app-nit-lifecycle.tex

```diff
@@ -0,0 +1,19 @@
+\documentclass[tikz]{standalone}
+
+\usetikzlibrary{automata,positioning}
+
+\begin{document}
+\begin{tikzpicture}[->,shorten >= 1pt,node distance=3cm,auto]
+	\node[state] (off)   {off};
```

Comment:
I think you can add `on_destroy` on the arrow between `off` and `stopped` even if it's specific to Android user.